<ui version="4.0" >
 <class>VP1CustomTourEditorForm</class>
 <widget class="QWidget" name="VP1CustomTourEditorForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Custom Tour</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QPushButton" name="pushButton_addCurrentView" >
     <property name="text" >
      <string>Add Current View</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>384</width>
        <height>345</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_utilityZoom" >
     <property name="title" >
      <string>Utility zoom</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" native="1" name="widget_utilityZoomContents" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Theta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_theta" >
           <property name="suffix" >
            <string> deg</string>
           </property>
           <property name="minimum" >
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>10.000000000000000</double>
           </property>
           <property name="value" >
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Radius:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_radius" >
           <property name="suffix" >
            <string> m</string>
           </property>
           <property name="minimum" >
            <double>0.010000000000000</double>
           </property>
           <property name="maximum" >
            <double>999.990000000000009</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_utilityZoomShow" >
           <property name="text" >
            <string>Show</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>1</width>
             <height>6</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_statustext" >
       <property name="text" >
        <string>Mixed camera types</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refreshPreviews" >
       <property name="text" >
        <string>&amp;Refresh previews</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_execute" >
       <property name="text" >
        <string>E&amp;xecute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox_utilityZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_utilityZoomContents</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>79</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
